grpc.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w( src/ruby/lib src/ruby/bin src/ruby/pb )
   s.platform      = Gem::Platform::RUBY
 
-  s.add_dependency 'google-protobuf', '~> 3.8'
+  s.add_dependency 'google-protobuf', '~> 3.9.0'
```

Comment:
I think the missing `.0` is intended. (see PR for bump to 3.8). Please fix in the .template file